File: tests/adapters/java/goldens/function_bodies/class_methods.java

public class Calculator {
    private List<String> history = new ArrayList<>();

    public Calculator(String name) {
        this.name = name;
        this.history = new ArrayList<>();
    }

    public int add(int a, int b) {
        // … method body omitted (3 lines)
    }

    public List<String> getHistory() {
        return new ArrayList<>(history);
    }
}

File: tests/adapters/kotlin/goldens/function_bodies/class_methods.kt

class Calculator(private val name: String) {
    private val history: MutableList<String> = mutableListOf()
    
    fun add(a: Int, b: Int): Int {
        // … method body omitted (3 lines)
    }
    
    fun getHistory(): List<String> {
        return history.toList()
    }
}
